### PR TITLE
Clarified ambiguous wording

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Unless you already knew what a "class parameter" is, it was totally ambiguous, which way is the recommended one.

### CODE REVIEW

**Comment by WouterJ** [comment]
To be honest, I don't see anything wrong with the original sentence. Your sentences seems to imply that using class parameters is recommended, but it surely isn't. Can you maybe elaborate your change a bit more?

**Comment by ThomasLandauer** [comment]
@WouterJ: The original sentence was "The previous example didn't do X:" which is ambiguous: Is the following code an example of still not doing X or an example of now doing X?
Since *you* certainly now what "Class Parameter" means, you probably can't see any difficulty ;-)
I know pulled the "cumbersome" sentence upwards to reveal the not-recommending point earlier.

File: best_practices/business-logic.rst [hunk 1]
```diff
@@ -136,8 +136,9 @@ of course use whatever format you like.
 Service: No Class Parameter
 ---------------------------
 
-You may have noticed that the previous service definition doesn't configure
-the class namespace as a parameter:
+This would be the way to define the service with the class namespace as a
+parameter (which is cumbersome and completely unnecessary for your own
+services):
```
Comment (xabbuh) [inline best_practices/business-logic.rst]: I would keep this as is.

File: best_practices/business-logic.rst [hunk 2]
```diff
@@ -151,8 +152,6 @@ the class namespace as a parameter:
         app.slugger:
             class: '%slugger.class%'
 
-This practice is cumbersome and completely unnecessary for your own services:
```
Comment (xabbuh) [inline best_practices/business-logic.rst]: We can instead reword this: "Using a parameter for the class name of your own services is cumbersome and completely unnecessary"

**Comment by ThomasLandauer** [comment]
OK, but please add this explanation in brackets:
"Using a parameter for the class name of your own services (as shown in the following code example) is cumbersome and completely unnecessary."

Your point is: Tell people that they should not do **it**.
My point is: You also need to explain what **it** means exactly.

**Comment by WouterJ** [comment]
I've just reread it again and I think it's clear enough. There is a comment in the code example indicating what is meant by "class namespaces as a parameter". Furthermore, the bestpractice admonition also clearly states the best practice.

I've fixed the little colon error in 5dfccb9. I'm going to close this PR. If you still don't agree, feel free to comment.

Thanks for trying to avoid confusing in the docs! I hope I'll be able to merge more PRs from you in the future :)

**Pull request closed without merging**